creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -94,19 +100,39 @@ const updateReplicaSetJobProcessor = async function ({
           log: true
         })
       healthyNodes = Object.keys(healthyServicesMap || {})
-      if (healthyNodes.length === 0)
+      if (healthyNodes.length === 0) {
+        result = 'failure_no_healthy_nodes'
         throw new Error(
           'Auto-selecting Content Nodes returned an empty list of healthy nodes.'
         )
+      }
       await cacheHealthyNodes(healthyNodes)
     } catch (e: any) {
+      if (result === 'success') {
+        result = 'failure_to_update_replica_set'
+      }
```

Comment:
don't think this accurately reflects the `try` block logic, maybe smth like `failure_find_healthy_nodes`